### PR TITLE
Release 3.8.1

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 3.8.1 (2020-05-06)
+
+- Fixed the list component to properly return to the first page when applying a filter or sorting. [#89](https://github.com/blackbaud/skyux-list-builder/pull/89)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md:3]: I'd remove properly. Also, does it return to the first page after you apply the filter? I'm inclined to say "..return to the first page after applying a filter...". Also, I'd add "a list" after sorting in the last sentence.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+- Fixed the list toolbar item component to properly honor item indexing. [#88](https://github.com/blackbaud/skyux-list-builder/pull/88)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md:4]: I'd remove properly.

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Done

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-list-builder?_sp.spa=pr-release-3.8.1](https://host.nxt.blackbaud.com/skyux-list-builder?_sp.spa=pr-release-3.8.1)

**Pull request merged**